src/microns_to_nwb/tools/ophys/ophys.py

```diff
@@ -140,7 +137,7 @@ def add_ophys(scan_key, nwb, timestamps):
             device=device,
             excitation_lambda=920.0,
             indicator="GCaMP6",
-            location=brain_areas,
+            location="VISp,VISrl,VISlm,VISal",
```

Comment:
I checked for the available sessions and scans the `AreaMembership` table where we can fetch the brain areas and they are the same, so simplified it without querying the actual areas and mapping them to these ontology names, but let me know if I should do it that way.

Reply:
Looks great!